tests/integration/test_deb_snap_migrate.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from pytest_operator.plugin import OpsTest
+import logging
+
+log = logging.getLogger(__name__)
+
+certs = [
+    "ETCDCTL_KEY_FILE=/var/snap/etcd/common/client.key",
+    "ETCDCTL_CERT_FILE=/var/snap/etcd/common/client.crt",
+    "ETCDCTL_CA_FILE=/var/snap/etcd/common/ca.crt",
+    "ETCDCTL_KEY=/var/snap/etcd/common/client.key",
+    "ETCDCTL_CERT=/var/snap/etcd/common/client.crt",
+    "ETCDCTL_CACERT=/var/snap/etcd/common/ca.crt",
+]
+
+
+@pytest.mark.abort_on_fail
+async def test_build_and_deploy(ops_test: OpsTest):
+    charm = await ops_test.build_charm(".")
+    await ops_test.model.deploy("cs:~containers/easyrsa-441")
+    await ops_test.model.deploy("cs:~containers/etcd-655")
```

Comment:
wait, you just built the etcd charm -- but your deploying from the charmstore?  did you mean:
```suggestion
    await ops_test.model.deploy(charm)
```

Reply:
This is basically a mirror of the old tests, the local build is used for charm refresh. I believe it is a kind of test, but not sure if it's a great test.

Reply:
i -- dunno but this looks wrong.  There's no need to deploy a FIXED version of the charm `cs:~containers/etcd-655` rather than the new charm we're trying to test in a PR.  Also we should deploy from `easyrsa` from charmhub with

```python
    await ops_test.model.deploy("easyrsa")
```